cartographer/mapping_2d/sparse_pose_graph.h

```diff
@@ -80,6 +81,11 @@ class SparsePoseGraph : public mapping::SparsePoseGraph {
   void AddImuData(int trajectory_id, const sensor::ImuData& imu_data);
   void AddOdometerData(int trajectory_id,
                        const sensor::OdometryData& odometry_data);
+  // Adds fixed frame pose data(like gps, pose, etc.) to be used in the
```

Comment:
Move comment? We should document this at a different place. Maybe where `sensor::FixedFramePoseData` is defined? Or at `mapping/trajectory_builder.h` which will eventually be the public API?